applications/Unity.GrantManager/src/Unity.GrantManager.Web/appsettings.json

```diff
@@ -13,6 +13,9 @@
     "IsJobExecutionEnabled": true,
     "Quartz": {
       "IsAutoRegisterEnabled": true
+    },
+    "IntakeResync": {
+      "Expression": "0 0 1 * * ?"
     }

```

Comment:
We need to set this up in dev config map and then let Daryl know to add these to some xml scripts in the Intake Resync expression

Reply:
I thought I could do this soley within our source code but there are environmental values he sets up outside of the yaml. It would be nice if we did not have to custom deploy env params
unity-grantmanager-web.yaml was where I thought this would happen but it sounded like from Daryl that there are some manual steps.
I guess the goal would be a zero touch deploy process

Reply:
![image](https://github.com/bcgov/Unity/assets/39963238/9c8649b1-866c-41e5-a023-7f6470c01b42)
For the Teams Notification I will update this branch to my new name. 
Since we were refactoring teams notification into the notification module I renamed the appsettings/config map